core/src/main/java/com/novoda/noplayer/Heart.java

```diff
@@ -30,6 +30,10 @@ public void startBeatingHeart() {
         handler.post(heartbeat);
     }
 
+    public void bind(Heartbeat onHeartbeat) {
```

Comment:
maybe me being picky but I'd prefer this method to be above `startBeatingHeart` as it has to happen first! 😸  
and _maybeeee_ an `IllegalStateException` if the `heartbeatAction` is null? What do you think?